README.md

```diff
@@ -57,33 +58,67 @@ To use this library in your project manually you may:
 1. for Projects, just drag BackgroundVideo.swift to the project tree
 2. for Workspaces, include the whole SwiftVideoBackground.xcodeproj
 
+## Migration Guide
+
+Version 2.0 brings improvements and breaking changes. See the quick migration guide [here]().
+
 ## Usage
 
-Import the framework into the ViewController
-```swift
-import SwiftVideoBackground
-```
+#### Example
 
-Link a UIView within a ViewController within the Storyboard to a BackgroundVideo item, or link it programmatically.
-```swift
-var backgroundVideo : BackgroundVideo!
-```
+``` swift
+import UIKit
+import VideoBackground
 
-Use the `createBackgroundVideo` function, with the name of the video or gif under `name`, and the file type under `type`. You can also include an alpha value between 0 and 1 under `alpha`, to adjust the brightness of the video.
-```swift
-backgroundVideo.createBackgroundVideo(name: "Background", type: "mp4")
-```
-```swift
-backgroundVideo.createBackgroundVideo(name: "Background", type: "mp4", alpha: 0.5)
+class MyViewController: UIViewController {
+  private let videoBackground = VideoBackground()
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    videoBackground.play(view: view, videoName: "myvideo", videoType: "mp4")
+  }
+}
 ```
 
-Note: Make sure you have added a video file to the project, and targeted the project. Also, make sure that you have set the module to `SwiftVideoBackground` for the BackgroundVideo UIView.
+> Documentation for Version 0.06 (Swift 3) can be found [here]().
+
+#### Customization Options
 
-To mute the video, simply set `isMuted` to true.
-```swift
-backgroundVideo.isMuted = true
+`play()` has three additional optional parameters:
+- `isMuted`: Bool - Indicates whether video is muted. Defaults to `true`.
+- `alpha`: CGFloat - Value between 0 and 1. The higher the value, the darker the video. Defaults to `0`.
+- `willLoopVideo`: Bool - Indicates whether video should restart when finished. Defaults to `true`.
+
+So for example:
+
+``` swift
+videoBackground.play(view: view,
+                     videoName: "myvideo",
+                     videoType: "mp4",
+                     isMuted: false,
+                     alpha: 0.25,
+                     willLoopVideo: true)
 ```
 
+-> will play the video with the sound on, slightly darkened, and will continuously loop.
+
```

Comment:
Will probably want to add something about the fact that you can add any combination of the three.